services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/util/DateTime.java

```diff
@@ -32,8 +32,10 @@
  * Methods for conversions of time/date data.
  */
 public class DateTime {
-
-  private DateTime() {
+  /**
```

Comment:
This comment is not necessary. Please remove in order to stay consistent with other similar private constructors in the code.

Reply:
change pushed, removed comment.